instagram/views.py

```
@@ -105,6 +132,25 @@ def get_queryset(self):
+    filter_backends = [filters.OrderingFilter]
+    ordering_fields = ["story_created_at", "created_at"]
+    ordering = ["-story_created_at"]
+
+    def get_queryset(self):
+        uuid = self.kwargs.get("uuid", None)
+        try:
+            user = InstagramUser.objects.get(uuid=uuid)
+            queryset = Story.objects.select_related("user").filter(user=user)
+            return queryset
+        except InstagramUser.DoesNotExist:
+            raise NotFound("Instagram user not found")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)